src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Authoring/Microsoft.Azure.CognitiveServices.LUIS.Authoring.csproj

```diff
@@ -6,10 +6,17 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring</PackageId>
     <Description>Provides API functions for consuming the Microsoft Azure Cognitive Services LUIS Authoring API.</Description>
-    <Version>0.9.0-preview</Version>
+    <Version>1.0.0</Version>
```

Comment:
Out of curiosity, all other SDKs are still preview, are you sure this is the only one you wish to mark as stable.